qiskit_experiments/curve_analysis/curve_analysis.py

```diff
@@ -856,11 +859,20 @@ def _data(
     def _arg_parse(self, **options) -> Dict[str, Any]:
         """Parse input kwargs with predicted input.
 
+        Class attributes will be updated according to the`options`.
```

Comment:
Nice documentation !

```suggestion
        Class attributes will be updated according to the ``options``.
```
same for other values, i.e. single back quote will become itaric (perhaps fine since this is private method, but just in case).

Reply:
I've committed a fix